packages/console/package.json

```diff
@@ -61,23 +61,23 @@
     "prettier": "^2.7.1",
     "process": "^0.11.10",
     "prop-types": "^15.8.1",
-    "react": "^17.0.2",
+    "react": "^18.0.0",
```

Comment:
18.2.0 was released on June 14. I think we should use the latest version

Reply:
see lockfile, it's using 18.2.0